docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -81,3 +82,5 @@
  go to [SkyAPM java plugin extension repository](https://github.com/SkyAPM/java-plugin-extensions) to get these.
 
 ²These plugins affect the performance or must be used under some conditions, from experiences. So only released in `/optional-plugins`, copy to `/plugins` in order to make them work.
+
+³JDK 8 Required. So only released in `/optional-plugins`, copy to `/plugins` in order to make them work.
```

Comment:
We don't need this. Option 2 is clear. I think you still miss `docs/en/setup/service-agent/java-agent/README.md#Optional Plugins`, could you recheck?